Makefile

```diff
@@ -368,8 +368,9 @@ version: mods/ra/mod.yaml mods/cnc/mod.yaml mods/d2k/mod.yaml mods/ts/mod.yaml m
 	done
 
 docs: utility mods version
-	@mono --debug OpenRA.Utility.exe all --docs > DOCUMENTATION.md
-	@mono --debug OpenRA.Utility.exe all --lua-docs > Lua-API.md
+	@mono --debug OpenRA.Utility.exe all --docs > TRAITS.md
+	@mono --debug OpenRA.Utility.exe all --lua-docs > LUA-API.md
+	@mono --debug OpenRA.Utility.exe all --settings-docs > Settings.md
```

Comment:
Having all three pages in regular case would also be fine by me.  The important point is that they agree.